frontend/ui/data/keyboardlayouts/keypopup/en_popup.lua

```diff
@@ -307,26 +349,27 @@ return {
         "ō",
         "ɔ", -- open o, open-mid back rounded vowel IPA
         "ɒ", -- turned alpha, open back rounded vowel IPA
+        "ö",
```

Comment:
> the small “o” keys has a lot of popups. There are four popups beyond compass directions, but because “o” is close to the right edge we don’t actually see the fourth symbol. Should we move it elsewhere?

Was wondering what you were talking about. I guess you mean this added one on line 352, which I don't see displayed at all - because we limit the extra ones to 3, so the 4th++ are ignored (they don't generate a 4th column in the popup).
So, I guess we have to sacrifice one of these, and probably not the `ö` which should be rather common if it were placed "north" previously.